boa/integrations/jupyter/browser.py

```diff
@@ -134,6 +124,31 @@ def wait_for_tx_receipt(self, tx_hash, timeout: float, poll_latency=1):
         )
 
 
+class BrowserEnv(NetworkEnv):
+    """
+    A NetworkEnv object that uses the BrowserSigner and BrowserRPC classes.
+    """
+
+    def __init__(self, address=None):
+        super().__init__(rpc=BrowserRPC())
```

Comment:
nit -- BrowserRPC() could have side effects, i'd personally put it on its own line --
```
rpc = BrowserRPC()
super().__init__(rpc=rpc)
```